net/pdns/files/pdns.conf-dist

```diff
@@ -140,6 +140,11 @@
 #
 # default-api-rectify=yes
 
+#################################
+# default-catalog-zone	Catalog zone to assign newly created primary zones (via the API) to
+#
+# default-catalog-zone=
```

Comment:
There isn't said anything about this in the commit description. You should do these changes in separate commit.

Reply:
I can update the commit description, but doing it in a separate commit doesn't make sense. It's part of the version bump.

Reply:
commit description updated